src/main/java/org/zalando/pazuzu/feature/FeatureRepositoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.pazuzu.feature;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.EntityManager;
+import java.util.List;
+
+public class FeatureRepositoryImpl implements FeatureRepositoryCustom {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Override
+    public List<Feature> getFeatures(int offset, int limit) {
```

Comment:
Yes, we have considered Pageable from Spring Data.
It works nice, is easy to use and additionally has sorting abilities.
Te following URL is used for pagination: /api/features?page=4&size=5
However, page/size parameters seem to violate Zalando guidelines http://zalando.github.io/restful-api-guidelines/pagination/Pagination.html
Briefly, the guidelines suggest to use one of the following:
1. cursor navigation
2. /api/features?offset=20&limit=5
None of them is compatible with Pageable.
But if we decide that we are not bounded by pagination guidelines, we can of course use Spring Data Pageable.